packages/components/src/components/contract-card/contract-card.tsx

```diff
@@ -54,6 +56,7 @@ const ContractCard = ({
                         is_visible={!!contract_info.is_sold}
                         onClickRemove={onClickRemove}
                         onClick={() => toggleUnsupportedContractModal?.(true)}
+                        payout_info={payout_info}
```

Comment:
@henry-deriv nit-pick :) if `payout_info` is used just once, we can put the template string here straight away

Reply:
applied changes